cogeo_mosaic/backends/dynamodb.py

```diff
@@ -0,0 +1,140 @@
+import functools
+import itertools
+import json
+import logging
+import os
+from decimal import Decimal
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import boto3
+import mercantile
+
+from cogeo_mosaic.backends.base import BaseBackend
+from cogeo_mosaic.backends.utils import find_quadkeys
+from cogeo_mosaic.model import MosaicJSON
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
🤔 I wonder if the logger shouldn't be set in the backend itself. right now I'm not sure `logger.warn` will show up later because we set it here with `INFO`

Reply:
I always just use print myself 😅 so I'll leave this to you